tests/test_frameSetUtils.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+from lsst.afw.geom import arcseconds, degrees, makeCdMatrix, Point2D, SpherePoint
+from lsst.afw.geom.detail import makeTanWcsMetadata, readFitsWcs, readLsstSkyWcs
+import lsst.utils.tests
+
+PrintStrippedNames = False
```

Comment:
You may be right but I'm going to resist this one. I hate uppercase with underscores.